File: labs-training-master/java/solid/solid-java-1/src/main/java/es/emergya/labs/training/solid_java_1/ocp/excercise_00/SaladIngredientsPackage.java
package es.emergya.labs.training.solid_java_1.ocp.excercise_00;

public class SaladIngredientsPackage implements IngredientsInterface {

private static String message = "Making salad with %.2fKg of lettuce, %.2fKg of tomatoes and %.2fKg of tuna.";
	
	private Float lettuceKg;
	private Float tomatoesKg;
	private Float tuneKg;

	public SaladIngredientsPackage(Float lettuceKg, Float tomatoesKg, Float tuneKg) {
		super();
		this.lettuceKg = lettuceKg;
		this.tomatoesKg = tomatoesKg;
		this.tuneKg = tuneKg;
	}

	public void make() {

		System.out.println(String.format(message, lettuceKg, tomatoesKg, tuneKg));
	}

	@Override
	public void makeFood() {
		make();
		
	}
	
}

File: labs-training-master/SolidPython/main/lsp/SolidPythonResuelto/main/ocp/SaladIngredientsPackage.py
from IngredientsPackageInterface import IngredientsPackageInterface

class SaladIngredientsPackage(IngredientsPackageInterface) :
    def __init__(self, lettuceKg:str,  tomatoesKg:str,  tuneKg:str) :
        self.lettuceKg = lettuceKg
        self.tomatoesKg = tomatoesKg
        self.tuneKg = tuneKg
        self.message:str = "Making salad with {0}Kg of lettuce, {1}Kg of tomatoes and {2}Kg of tuna.".format(str(lettuceKg),str(tomatoesKg),str(tuneKg))
    def make(self) -> None:
        print(self.message)